src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { cli, command, error } from './waterfall-cli';
+export default cli;
+export { command, error, cli };
```

Comment:
Why do we also export `cli` here, if it's already being exported as the default?

Reply:
I did this intentionally, so that the default would be available also as a named export, so that clients would have some additional flexibility in how the module is consumed.

Reply:
I would prefer we export it one way only; otherwise, we'll have clients using this codebase in inconsistent ways. This would make updating the codebase more likely to produce breaking changes and make intentionally breaking changes harder to communicate. It would also make the documentation surrounding how to use this functionality more complex.